test/misc.jl

```diff
@@ -28,6 +28,99 @@ let bt = backtrace()
     end
 end
 
+# sprint() with error() throws an error, but not with info() or warn().  why?
+module Foo
+    function bar(io)
+        info(io,"barinfo")
+        warn(io,"barwarn")
+        #error(io,"barerror")
+    end
+    function pooh(io)
+        info(io,"poohinfo")
+        warn(io,"poohwarn")
+        #error(io,"pooherror")
+    end
+end
+function foo(io)
+    info(io,"fooinfo")
+    warn(io,"foowarn")
+    #error(io,"fooerror")
+end
+
+@test ( tmp = sprint(Foo.bar);
+        all(map(x->contains(tmp,x),
```

Comment:
could you not? the `@test ( tmp = ...; ... )` doesn't fit with the style used by most existing tests